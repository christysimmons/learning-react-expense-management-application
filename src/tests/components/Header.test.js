import React from 'react';
import { shallow } from 'enzyme';
//import ReactShallowRenderer from 'react-test-renderer/shallow';
import Header from '../../components/Header';

test('should load header correctly', () => {
    //Method for enzyme
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
    
    
    //expect(wrapper.find('h1').text()).toBe('Expension');
    /*
    Method for react-test-renderer

    const renderer = new ReactShallowRenderer();
    renderer.render(<Header />);
    expect(renderer.getRenderOutput()).toMatchSnapshot(); 
    */

});