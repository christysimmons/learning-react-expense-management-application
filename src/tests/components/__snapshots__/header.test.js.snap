// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should load header correctly 1`] = `
<header
  className="header"
>
  <div
    className="content-container header__content"
  >
    <Link
      className="header__title"
      exact={true}
      to="/dashboard"
    >
      <h1>
        Expension
      </h1>
    </Link>
    <button
      className="button button--link"
      onClick={[Function]}
    >
      Logout
    </button>
  </div>
</header>
`;
